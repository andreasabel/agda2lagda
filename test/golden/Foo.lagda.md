<!-- This file was automatically generated by agda2lagda 0.2023.3.25. -->

Sample non-literate Agda program
================================

A remark to test bulleted lists:

* This file serves as example for agda2lagda.

* The content may be non-sensical.

Indeed!

```agda
module Foo where
```

Some data type.

```agda
data D : Set where
  c : D
```

A function.

```agda
foo : D → D
foo c = c   -- basically, the identity
```

<!-- This part is commented out.
{-
bar : D → Set
bar x = D
-- -}
--
 -->
A subheading
-------------

```agda
module Submodule where

  postulate
    zeta : D
```

That's it.
Bye.
