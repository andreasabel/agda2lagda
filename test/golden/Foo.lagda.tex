%% This file was automatically generated by agda2lagda 0.2021.6.1.

\heading{Sample non-literate Agda program}

A remark to test bulleted lists:

\begin{itemize}

\item
This file serves as example for agda2lagda.

\item
The content may be non-sensical.

\end{itemize}

Indeed!

\begin{code}
module Foo where
\end{code}

Some data type.

\begin{code}
data D : Set where
  c : D
\end{code}

A function.

\begin{code}
foo : D → D
foo c = c   -- basically, the identity
\end{code}

%% This part is commented out.
%% {-
%% bar : D → Set
%% bar x = D
%% -- -}
%% --

\subheading{A subheading}

\begin{code}
module Submodule where

  postulate
    zeta : D
\end{code}

That's it.
Bye.

